### PR TITLE
Update for v2

### PR DESCRIPTION
**Pull request by mottox2**
refs gatsbyjs/gatsby#5973

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 
 const IndexPage = () => (
-  <div>
-    <h1>Hi people</h1>
-    <p>Welcome to your new Gatsby site.</p>
-    <p>Now go build something great.</p>
-    <Link to="/page-2/">Go to page 2</Link>
-  </div>
+  <Layout>
+    <>
```
Comment (jlengstorf) [inline src/pages/index.js]: No need for this fragment; the `<Layout>` should work fine as a wrapper.

File: src/pages/page-2.js [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
+import Layout from '../components/layout'
 
 const SecondPage = () => (
-  <div>
-    <h1>Hi from the second page</h1>
-    <p>Welcome to page 2</p>
-    <Link to="/">Go back to the homepage</Link>
-  </div>
+  <Layout>
+    <>
```
Comment (jlengstorf) [inline src/pages/page-2.js]: Same here — no need for the fragment.

**Comment by 0x6e6562** [comment]
Whats the best way to use this starter with gatsby-cli?

**Comment by 0x6e6562** [comment]
I think this might be the solution:

`$ gatsby new YOUR_DIRECTORY 'https://github.com/gatsbyjs/gatsby-starter-default.git#v2'`

**Comment by jlengstorf** [comment]
That's the right solution! You don't need the .git at the end, though. It
works either way.


-- 

*Jason Lengstorf*
Find me online: *Twitter <https://twitter.com/jlengstorf>* · GitHub
<https://github.com/jlengstorf> · *LinkedIn
<https://linkedin.com/in/jlengstorf/>*
Oversharing at *lengstorf.com <https://lengstorf.com/>*

**Pull request merged**